omod/src/main/webapp/portlets/patientPrograms.jsp

```diff
@@ -129,10 +129,6 @@
 				alert('<openmrs:message code="State.error.invalidDate" javaScriptEscape="true"/>');
 				return;
 			}
-			if (!isEmpty(lastStateEndDate)) {
```

Comment:
Did you also intentionally remove this?

Reply:
Yes. 
It is frequent to have patients that exit/default from care and they return.
It is easier to resume care and edit the latest states.
This allows use to edit the state too if they exit the patient from care (...and closed the states) by mistake